core/domain/question_services_test.py

```diff
@@ -36,6 +38,59 @@ def setUp(self):
         user_services.create_new_user(self.owner_id, self.OWNER_EMAIL)
         self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
 
+    def test_get_question_by_id(self):
+        state = exp_domain.State.create_default_state('ABC')
+        question_data = state.to_dict()
+        question_id = 'dummy'
+        title = 'A Question'
+        question_data_schema_version = 1
+        collection_id = 'col1'
+        language_code = 'en'
+        question = question_domain.Question(
```

Comment:
Here and below, for the variables you're only using once, just inline it in this call. Easier to read -- fewer things to keep track of.

Reply:
Updated.